packages/hash/frontend/src/blocks/page/Comments/CommentThread.tsx

```diff
@@ -73,6 +73,11 @@ export const CommentThread: FunctionComponent<CommentThreadProps> = ({
     [comment.author.properties],
   );
 
+  const authorId = useMemo(
+    () => comment.author.metadata.editionId.baseId.split("%")[1],
```

Comment:
Is there a helper function/better way to get this?

Reply:
packages/hash/subgraph/src/types/identifier.ts:27

`extractOwnedByIdFromEntityId`

Reply:
Thank you! Fixed in fffd735ae33f2407da48ad82fe5a791edd9f7285.